.github/workflows/build-and-release.yml

```
@@ -0,0 +1,54 @@
+name: Build and Release
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types: [created]
```

from GovernanceExpert
The workflow is triggered on every push to the main branch and on release creation. This might not align with governance policies if every push is not intended to be built and released. Consider adding conditions or filters to ensure only relevant changes trigger the workflow.

```suggestion
on:
  push:
    branches:
      - main
    paths:
      - 'src/**'
  release:
    types: [created]
```